### PR TITLE
Define new READMEs for Docker Hub

### PR DESCRIPTION
**Pull request by mthalman**
In order to address the character limit issue with #5327, these changes define a separate set of READMEs that are specific to Docker Hub. These READMEs are configured to have the tags tables removed and instead include links to the MAR portal and this repo.

Contributes to #5327

### CODE REVIEW

File: .portal-docs/README.aspire-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+* [dotnet/nightly/runtime](https://hub.docker.com/_/microsoft-dotnet-nightly-runtime/): .NET Runtime (Preview)
+* [dotnet/nightly/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-nightly-runtime-deps/): .NET Runtime Dependencies (Preview)
+* [dotnet/nightly/monitor](https://hub.docker.com/_/microsoft-dotnet-nightly-monitor/): .NET Monitor Tool (Preview)
+
+.NET Framework:
+
+* [dotnet/framework](https://hub.docker.com/_/microsoft-dotnet-framework/): .NET Framework, ASP.NET and WCF
+* [dotnet/framework/samples](https://hub.docker.com/_/microsoft-dotnet-framework-samples/): .NET Framework, ASP.NET and WCF Samples
+
+# Full Tag Listing
```
Comment (MichaelSimons) [inline .portal-docs/README.aspire-dashboard.md:111]: From the PR description, I thought this was being removed?

**Review by MichaelSimons — COMMENTED** [review]
Is there a way we can make the portal-docs apparent what portal (mcr/dh) they are for?  Either sub-folder or a filename prefix/suffix?

File: .portal-docs/README.aspire-dashboard.md (hunk 1 above)
Comment (MichaelSimons) [reply .portal-docs/README.aspire-dashboard.md:111]: > The table is being removed.

But the table is still present.  I was under the impression it is being removed with this PR.

Comment (MichaelSimons) [reply .portal-docs/README.aspire-dashboard.md:111]: see https://github.com/dotnet/dotnet-docker/pull/5585/files#diff-eebee3282380281b727711c740ab97298dc872b487bd13e547490bd78e5d2898R113

Comment (mthalman) [reply .portal-docs/README.aspire-dashboard.md:111]: Oh, what the heck. There's another `Full Tag Listing` header further below that is configured correctly: https://github.com/dotnet/dotnet-docker/pull/5585/files#diff-eebee3282380281b727711c740ab97298dc872b487bd13e547490bd78e5d2898R233. I need to figure out what's going on.

**Comment by mthalman** [comment]
> Is there a way we can make the portal-docs apparent what portal (mcr/dh) they are for? Either sub-folder or a filename prefix/suffix?

The filenames need to be exactly what they are. That's what MAR requires. Yes, technically they could be in separate directories but that would require a different implementation than what was provided by https://github.com/dotnet/docker-tools/pull/1345.

**Review by MichaelSimons — APPROVED** [review]
Approving to unblock the readme issue in DH.  I am not a fan of the readme file layout.  It is difficult for outsiders/new contributors to understand the differences in the two types of readmes.  I think this is something that should be fixed in a follow-up.

File: .portal-docs/README.runtime.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* [dotnet/nightly/aspire-dashboard](https://hub.docker.com/_/microsoft-dotnet-nightly-aspire-dashboard/): .NET Aspire Dashboard (Preview)
+
+.NET Framework:
+
+* [dotnet/framework](https://hub.docker.com/_/microsoft-dotnet-framework/): .NET Framework, ASP.NET and WCF
+* [dotnet/framework/samples](https://hub.docker.com/_/microsoft-dotnet-framework-samples/): .NET Framework, ASP.NET and WCF Samples
+
+# Full Tag Listing
+
+View the current tags at the [Microsoft Artifact Registry portal](https://mcr.microsoft.com/product/$dotnet/nightly/runtime/tags) or on [Docker Hub](https://github.com/dotnet/dotnet-docker/blob/nightly).
```
Comment (MichaelSimons) [inline .portal-docs/README.runtime.md:66]: Shouldn't this say GitHub?
```suggestion
View the current tags at the [Microsoft Artifact Registry portal](https://mcr.microsoft.com/product/$dotnet/nightly/runtime/tags) or on [GitHub](https://github.com/dotnet/dotnet-docker/blob/nightly).
```

Comment (MichaelSimons) [reply .portal-docs/README.runtime.md:66]: Also, Can we should deep link to the appropriate readme's Full Tag Listing section.

Comment (MichaelSimons) [reply .portal-docs/README.runtime.md:66]: One more thing - the MAR link appears broken because of the `$dotnet`.

**Pull request merged**